source/lib/include/ppp/project/project.hpp

```diff
@@ -42,6 +42,16 @@ enum class CardCorners
     Rounded,
 };
 
+struct FourMargins
```

Comment:
I've been toying a bit with having only one custom margins object in the project that looks like this
```cpp
struct CustomMargins
{
    Size m_TopLeftMargins{ 0_mm, 0_mm };
    std::optional<Size> m_BottomRightMargins{ std::nullopt };
};
```
So we only have one entry point for margins, not two different functions (`ComputeMargins` and `ComputeMarginsFour`). And that seemed to simplify user-code.

What do you think?